Formula/httest.rb

```diff
@@ -15,11 +16,17 @@ class Httest < Formula
   depends_on "apr-util"
   depends_on "openssl"
   depends_on "pcre"
-  depends_on "lua" => :recommended
+  depends_on "lua@5.1" => :recommended
   depends_on "nghttp2" => :recommended
   depends_on "spidermonkey" => :recommended
 
+  deprecated_option "without-lua" => "without-lua@5.1"
+
   def install
+    inreplace "configure",
+      "else LUA_LIB_PATH=\"-L${withval}\"; LUA_INCLUDES=\"-I${withval}\"; LUA_LIB=\"-llua\"; fi",
```

Comment:
They've hardcoded expecting a vanilla `liblua`? 🤦‍♂️.

Reply:
It's gibberish. Notice they expect to find the library and the includes in the exact same directory without taking into account include vs. lib subdirectories.

Here's how it searches:
```
# Check whether --with-lua was given.
if test "${with_lua+set}" = set; then
  withval=$with_lua; if test ! -d $withval; then { { $as_echo "$as_me:$LINENO: error: $withval is not a directory" >&5
$as_echo "$as_me: error: $withval is not a directory" >&2;}
   { (exit 1); exit 1; }; }; else LUA_LIB_PATH="-L${withval}"; LUA_INCLUDES="-I${withval}"; LUA_LIB="-llua"; fi
else
  LUA_LIB_PATH=""; if test -d /usr/include/lua5.1; then LUA_INCLUDES="-I/usr/include/lua5.1"; else LUA_INCLUDES=""; fi; if test -f /usr/lib/liblua5.1.a -o -f /usr/lib/liblua5.1.so -o -f /usr/lib/i386-linux-gnu/liblua5.1.so -o -f /usr/lib/i386-linux-gnu/liblua5.1.a; then LUA_LIB="-llua5.1"; else LUA_LIB="-llua"; fi
fi
```

Reply:
That is... that is special.